src/proto/Ethereum.proto

```diff
@@ -42,8 +42,11 @@ message Transaction {
 
     // Generic smart contract transaction
     message ContractGeneric {
+        // Amount to send in wei (256-bit number)
+        bytes amount = 1;
+
```

Comment:
In contract calls, amount is always 0.  I think this field is not needed, confusing.

Reply:
@leoneparise suggests add it

Reply:
normal transfer or contract call can both have amount and data:

simple transfer:
https://etherscan.io/tx/0xe15e3234a7e665793d8c6570026b9ecbb18e123edade2d2f1a7053acdc72a29e

contract call:
https://bscscan.com/tx/0x117caac3a24d7c17a55b3b1b7d84539668d9222809e9f014723b0960ed57f250